roles/openjdk/vars/main.yml

```
@@ -1,7 +1,7 @@
 ---
 # vars file for roles/openjdk
 _java_default_version:
-  Ubuntu-19: 13
+  Ubuntu-19: 17
```

This seems merge-able if we don't update the default version:
```suggestion
  Ubuntu-19: 13
```